lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -257,7 +266,11 @@ TFLiteModelLoader::getTensorByIndex(size_t index) {
 }
 
 std::string TFLiteModelLoader::getTensorName(const tflite::Tensor *tensor) {
-  return tensor->name()->str();
+  if (tfliteRenameTensorsOpt) {
+    return std::string("tensor") + std::to_string((size_t)(tensor));
```

Comment:
I guess the only issue it will cause that on each new run tensor names will be different.